mscore/propertymenu.cpp

```diff
@@ -524,6 +493,45 @@ void ScoreView::editTimeSigProperties(TimeSig* ts)
       delete r;
       }
 
+//---------------------------------------------------------
+//   selectInstrument
+//---------------------------------------------------------
+
+void Ms::ScoreView::selectInstrument(InstrumentChange* ic)
+      {
+      SelectInstrument si(ic->instrument(), 0);
+      if (si.exec()) {
+            const InstrumentTemplate* it = si.instrTemplate();
+            if (it) {
+                  Fraction tickStart = ic->segment()->tick();
+                  Part* part = ic->staff()->part();
+                  Interval oldV = part->instrument(tickStart)->transpose();
+                  //Instrument* oi = ic->instrument();  //part->instrument(tickStart);
+                  //Instrument* instrument = new Instrument(Instrument::fromTemplate(it));
+                  // change instrument in all linked scores
+                  for (ScoreElement* se : ic->linkList()) {
+                        InstrumentChange* lic = toInstrumentChange(se);
+                        Instrument* instrument = new Instrument(Instrument::fromTemplate(it));
+                        lic->score()->undo(new ChangeInstrument(lic, instrument));
+                        }
+                  // transpose for current score only
+                  // this automatically propagates to linked scores
+                  if (part->instrument(tickStart)->transpose() != oldV) {
+                        auto i = part->instruments()->upper_bound(tickStart.ticks());    // find(), ++i
+                        Fraction tickEnd;
+                        if (i == part->instruments()->end())
+                              tickEnd = Fraction(-1, 1);
+                        else
+                              tickEnd = Fraction::fromTicks(i->first);
+                        ic->score()->transpositionChanged(part, oldV, tickStart, tickEnd);
+                        }
+                  }
+            else
+                  qDebug("no template selected?");
+            }
+      }
+}
```

Comment:
This brace is probably not needed: `Ms` namespace scope is closed below at line 554. This is probably what causes build error, see https://travis-ci.org/github/musescore/MuseScore/jobs/663596083#L2701

Reply:
Oops, rebasing error, thanks